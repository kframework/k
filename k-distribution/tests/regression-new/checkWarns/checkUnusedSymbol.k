module CHECKUNUSEDSYMBOL-SYNTAX

endmodule

module CHECKUNUSEDSYMBOL
  imports CHECKUNUSEDSYMBOL-SYNTAX
  imports INT

  syntax Foo ::= foo(Int)
endmodule
