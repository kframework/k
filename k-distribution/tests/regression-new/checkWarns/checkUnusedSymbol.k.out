[Error] Compiler: Had 1 structural errors.
[Error] Compiler: Symbol 'foo(_)_CHECKUNUSEDSYMBOL_Foo_Int' defined but not
used.
	Source(checkUnusedSymbol.k)
	Location(9,18,9,25)
